sentry-android-core/src/main/java/io/sentry/android/core/AnrIntegration.java

```diff
@@ -41,8 +41,8 @@ private void register(IHub hub, SentryAndroidOptions options) {
                 Mechanism mechanism = new Mechanism();
                 mechanism.setType("ANR");
                 mechanism.setHandled(false);
-                ExceptionMechanismThrowable throwable =
-                    new ExceptionMechanismThrowable(mechanism, error, Thread.currentThread());
+                ExceptionMechanism throwable =
```

Comment:
We have it in the protocol the `ExceptionMechanism` which is called `Mechanism`. In C# it's actually called `ExceptionMechanism` so this name is a bit confusing.

Maybe MechanismException if this is in fact an `Exception`?

Reply:
ExceptionMechanismException now, so same pattern as before